src/frontend/src/pages/CreateChangeRequestPage/CreateChangeRequestView.tsx

```diff
@@ -122,6 +124,8 @@ const CreateChangeRequestsView: React.FC<CreateChangeRequestViewProps> = ({
     });
   });
 
+  wbsDropdownOptions.sort((a, b) => wbsNumComparator(a.id, b.id));
```

Comment:
we should also rename these from a and b to wbsNum1, wbsNum2 to be more precise/accurate